docs/buildah-from.md

```diff
@@ -46,13 +46,13 @@ If an image needs to be pulled from the registry, suppress progress output.
 
 ## EXAMPLE
 
-buildah from imagename --pull --registry "myregistry://"
+buildah from imagename --pull --transport "docker://myregistry.example.com/"
```

Comment:
Both should provide equivalent results, but the `--transport` option is necessary for `build-using-dockerfile` anyway.